modules/ossm-installing-multi-primary-network-mesh.adoc

```diff
@@ -0,0 +1,303 @@
++
+[source,terminal]
+----
+$ oc apply -f istio-east.yaml
+----
+
+.. Wait for the control plane to return the "Ready" status condition by running the following command:
++
+[source,terminal]
+----
+$ oc wait --context "${CTX_CLUSTER1}" --for condition=Ready istio/default --timeout=3m
+----
+
+.. Create an East-West gateway on the East cluster by running the following command:
++
+[source,terminal]
+----
+$ oc --context "${CTX_CLUSTER1}" apply -f https://raw.githubusercontent.com/istio-ecosystem/sail-operator/main/docs/multicluster/east-west-gateway-net1.yaml
+----
+
+.. Expose the services through the gateway by running the following command:
++
+[source,terminal]
+----
+$ oc --context "${CTX_CLUSTER1}" apply -n istio-system -f https://raw.githubusercontent.com/istio-ecosystem/sail-operator/main/docs/multicluster/expose-services.yaml
+----
+
+. Install {istio} on the West cluster:
+
+.. Create a YAML file named `istio-west.yaml` that defines an `Istio` resource on the West cluster.
++
+.Example resource file
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: sailoperator.io/v1alpha1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: v${ISTIO_VERSION}
+  namespace: istio-system
+  values:
+    global:
+      meshID: mesh1
+      multiCluster:
+        clusterName: cluster2
+      network: network2
+----
+
+.. Apply the YAML file by running the following command:
++
+[source,terminal]
+----
+oc --context "${CTX_CLUSTER2}" apply -f istio-west.yaml
+----
+
+.. Wait for the control plane to return the "Ready" status condition by running the following command:
++
+[source,terminal]
+----
+$ oc wait --context "${CTX_CLUSTER2}" --for condition=Ready istio/default --timeout=3m
+----
+
+.. Create an East-West gateway on the West cluster by running the following command:
++
+[source,terminal]
+----
+$ oc --context "${CTX_CLUSTER2}" apply -f https://raw.githubusercontent.com/istio-ecosystem/sail-operator/main/docs/multicluster/east-west-gateway-net2.yaml
+----
+
+.. Expose the services through the gateway by running the following command:
++
+[source,terminal]
+----
+$ oc --context "${CTX_CLUSTER2}" apply -n istio-system -f https://raw.githubusercontent.com/istio-ecosystem/sail-operator/main/docs/multicluster/expose-services.yaml
+----
+
+. Install a remote secret on the East cluster that provides access to the API server on the West cluster by running the following command:
++
+[source,terminal]
+----
+$ istioctl create-remote-secret \
+  --context="${CTX_CLUSTER2}" \
+  --name=cluster2 | \
+  oc --context="${CTX_CLUSTER1}" apply -f -
+----
+
+. Install a remote secret on the West cluster that provides access to the API server on the East cluster by running the following command:
++
+[source,terminal]
+----
+$ istioctl create-remote-secret \
+  --context="${CTX_CLUSTER1}" \
+  --name=cluster1 | \
+  oc --context="${CTX_CLUSTER2}" apply -f -
+----
+////
+. Deploy sample applications on the East cluster:
```

Comment:
Wasn't this moved to a different file?

Reply:
Yes, but I commented it out in this one just to make sure everything built cleanly before I deleted this. I'mm deleting it on the next push later today.